routes/teams.js

```diff
@@ -16,12 +16,15 @@ var golf_key = 'wuczn4z2ktufacuae7u8sxfc';
 
 
 var express = require('express');
+var request = require('request');
 var router = express.Router();
 var http = require("http"),
     mongojs = require("mongojs"),
-    uri = 'mongodb://root:root@ds031541.mongolab.com:31541/rosterblitz';
+    uri = 'mongodb://root:root@ds031541.mongolab.com:31541/rosterblitz',
+
+    db = mongojs.connect(uri, ["teams"]),
+    db_players = mongojs.connect(uri, ["players"]);
```

Comment:
We don't need two variables for this, since the collection is specified in the open function. Eliminate the 2nd param, and just with the db variable